### PR TITLE
Clements jsCalc

### PR DESCRIPTION
**Pull request by sdkclements**
Comfort: 4
completeness: 3.5
logic works, just need to set up the callbacks for enter and click to retrieve that values
_will update_

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -1,2 +1,114 @@
-var calculator = {
+// $(document).ready(function() {
+
+var calc = {
+    add: (num1, num2) => num1 + num2,
+    subt: (num1, num2) => num1 - num2,
+    mult: (num1, num2) => num1 * num2,
+    divide: (num1, num2) => num1 / num2,
+    expon: (num, exp) => Math.pow(num, exp),
```
Comment (amaseda) [inline calculator.js:8]: Can also use the `num**exp` syntax instead of `Math.pow` if you're into that.

File: index.html [hunk 2]
```diff
@@ -3,8 +3,45 @@
...
+      </div>
+    </div>
+    <script
+    src="https://code.jquery.com/jquery-3.1.1.js"
+    integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA="
+    crossorigin="anonymous"></script>
     <script src="calculator.js" charset="utf-8"></script>
   </body>
-</html>
+</html>
```
Comment (amaseda) [inline index.html:47]: :+1: For giving it some visuals.

**Comment by amaseda** [comment]
:+1: Let me know if you have any questions / comments after updating.

File: calculator.js (hunk 1 above)
Comment (sdkclements) [reply calculator.js:8]: I did that originally but instructors said that was bad

**Pull request closed without merging**